cloud-optimized-geotiffs/cogs-details.qmd

```diff
@@ -0,0 +1,41 @@
+---
+title: "Make better GeoTIFFs"
+---
+
+# Make better GeoTIFFs 
```

Comment:
All the rest of our sidebar titles and h1 headers are nouns instead of verbs; this reads a little weird to me.

Maybe `Advanced COG Details` or `Advanced COG Recommendations`? Each of our pages could have `Advanced *`. It also seems nice to have the title in the sidebar be the same or roughly the same as the URL fragment